src/DropDownSort/components/DropDownSortContainer.ts

```diff
@@ -41,7 +41,7 @@ export interface FormState {
 
 export default class DropDownSortContainer extends Component<ContainerProps, ContainerState> {
     private dataSourceHelper?: DataSourceHelper;
-    private widgetDom: HTMLElement | null = null;
+    private widgetDom: Element | null = null;
```

Comment:
I think the function "connectToListView" should also be changed over here.

Reply:
Not sure what you try to say? Are you referring to the error state fix? as this widget does not depend on context validation. it is not needed